modules/ROOT/pages/as2/AS2-connector-studio-6-0.adoc

```diff
@@ -0,0 +1,61 @@
+= AS2 - Studio Configuration - Mule 4
```

Comment:
This is all boilerplate text, isn't it?
I feel as if I have edited it many times.